modules/apps/29-fee/keeper/keeper_test.go

```diff
@@ -40,7 +40,7 @@ func (suite *KeeperTestSuite) SetupTest() {
 	suite.chainB = suite.coordinator.GetChain(ibctesting.GetChainID(1))
 
 	path := ibctesting.NewPath(suite.chainA, suite.chainB)
-	feeTransferVersion := channeltypes.MergeChannelVersions(types.Version, transfertypes.Version)
+	feeTransferVersion := string(types.ModuleCdc.MustMarshalJSON(&types.Metadata{Version: types.Version, AppVersion: transfertypes.Version}))
```

Comment:
maybe we should add a `MustMarshalJSON` function to Metadata? or a function to return the marshaled metadata as a string (unsure what a good name would be)

Reply:
Very much agree to be honest, this was on my mind when I was writing this. I'll think on it and add it in a follow up PR. It does start to look ugly seeing all the string conversions scattered around, I would prefer to just call some method on `Metadata` to do the job.